index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css">
+    <link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles.css" class="src">
+    <script src="index.js"></script>
+    <title>Porfolio</title>
+</head>
+<body>
+    <div class="main">
+      <header>
+        <div class="top-navigation">
+          <div class="wrapper">
+            <nav>
+              <ul>
+                <li class="name"><span>pius ssekweyama</span><i class="fa fa-bars mobile-menu" aria-hidden="true"></i></li>
+                <li class="desktop-menu">portfolio</li>
+                <li class="desktop-menu">about</li>
+                <li class="desktop-menu">contact</li>
+                <li class="social desktop-menu">
+                  <i class="fa fa-envelope" aria-hidden="true"></i>
+                </li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </header>
+
+
+      <div class="headline-section">
+        <div class="illustration-header"></div>
+        <div class="social-icons">
+          <i class="fa fa-github fa-lg" aria-hidden="true"></i>
+          <i class="fa fa-linkedin-square fa-lg" aria-hidden="true"></i>
+          <i class="fa fa-twitter fa-lg" aria-hidden="true"></i>
```

Comment:
kindly, I suggest following the advice of the previous reviewer and use UL/LI lists to organize the icons